.github/workflows/linters.yml

```diff
@@ -24,7 +24,7 @@ jobs:
       - run: sudo npm install --global npm@7
       - run: npm ci
       - run: npx prettier --check .
-      - run: npx tsc --noEmit
+      - run: npx tsc -b
```

Comment:
why?

Reply:
Because `--noEmit` doesn't work in project reference